src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -5,6 +5,14 @@
 from software.thunderscope.common import common_widgets
 from proto.import_all_protos import *
 from software.thunderscope.proto_unix_io import ProtoUnixIO
+from enum import IntEnum
+
+
+class DriveMode(IntEnum):
```

Comment:
The terms "drive mode" and "control mode" seem to be used interchangeably throughout the code. It's probably better to stick to one to keep things consistent